app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -182,11 +181,11 @@ const getActualAcuantJavascriptWebSdk = (): AcuantJavascriptWebSdkInterface => {
   if (window.AcuantJavascriptWebSdk && typeof window.AcuantJavascriptWebSdk.start === 'function') {
     return window.AcuantJavascriptWebSdk;
   }
-  if (typeof AcuantJavascriptWebSdk === 'undefined') {
+  if (!window.AcuantJavascriptWebSdk) {
     // eslint-disable-next-line no-console
     console.error('AcuantJavascriptWebSdk is not defined in the global scope');
```

Comment:
Not in scope of what you'd aimed to tackle, but I wonder why we log a message here. If we wanted to log something for us to be aware of, I'd think we'd use [`trackError`](https://github.com/18F/identity-idp/blob/8f74bcd1bd84b0e39a4b6d5f6f7df17cc7c2cc49/app/javascript/packages/analytics/index.ts#L23-L29). Or if it's for tests (to trip the [browser log detection](https://github.com/18F/identity-idp/blob/8f74bcd1bd84b0e39a4b6d5f6f7df17cc7c2cc49/spec/rails_helper.rb#L139-L152)), I'd think we could wrap that in `process.env.NODE_ENV`. If it's for local development, I wouldn't think we should have it part of the committed code.

Reply:
That's a good point. I'm not sure what the original intent of this was either, tbh. It was added almost 2 years ago and I don't see a direct reference to the "why" for the console error.

I think reading through it, my take on it would be that we'd want to know if it's happening in production. The user would probably be encountering some kind of error already if there is no `window.AcuantJavascriptWebSdk`. And I think we'd want to know if that's happening to users (especially if it's happening to many).

So I think `TrackError` makes sense. (TIL `trackError` exists! I thought we always used `trackEvent`). I created [LG-12733](https://cm-jira.usa.gov/browse/LG-12733) to track that work. I found we use it twice in this file as well.